### PR TITLE
[Fix](Nereids)fix loading core when enable nereids DML default

### PR DESCRIPTION
**Pull request by sohardforaname**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.14 seconds
 stream load tsv:          543 seconds loaded 74807831229 Bytes, about 131 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.6 seconds inserted 10000000 Rows, about 337K ops/s
 storage size: 17162309157 Bytes

**Comment by sohardforaname** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.9 seconds
 stream load tsv:          541 seconds loaded 74807831229 Bytes, about 131 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17162201054 Bytes

**Comment by sohardforaname** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.35 seconds
 stream load tsv:          539 seconds loaded 74807831229 Bytes, about 132 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.4 seconds inserted 10000000 Rows, about 340K ops/s
 storage size: 17162110346 Bytes

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.53 seconds
 stream load tsv:          547 seconds loaded 74807831229 Bytes, about 130 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.8 seconds inserted 10000000 Rows, about 335K ops/s
 storage size: 17162103944 Bytes

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java [hunk 1]
```diff
@@ -226,6 +233,17 @@ private List<Column> bindTargetColumns(OlapTable table, List<String> colsName) {
                 }).collect(Collectors.toList());
     }
 
+    private void maybeFallBackAggStateTypeColumn(Column column, ConnectContext ctx) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java:236]: i think u should add aggstate type into `org.apache.doris.nereids.rules.rewrite.CheckDataTypes#UNSUPPORTED_TYPE`

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/CreateTableCommand.java [hunk 2]
```diff
@@ -107,7 +107,7 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
             } else if (i == 0 && dataType.isStringType()) {
                 dataType = VarcharType.createVarcharType(ScalarType.MAX_VARCHAR_LENGTH);
             }
-            columnsOfQuery.add(new ColumnDefinition(s.getName(), dataType, s.nullable()));
+            columnsOfQuery.add(new ColumnDefinition(s.getName(), dataType, !s.isColumnFromTable() || s.nullable()));
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/CreateTableCommand.java:110]: add comment to explain it

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/UpdateCommand.java [hunk 3]
```diff
@@ -105,7 +105,7 @@ public LogicalPlan completeQueryPlan(ConnectContext ctx, LogicalPlan logicalQuer
         List<NamedExpression> selectItems = Lists.newArrayList();
         String tableName = tableAlias != null ? tableAlias : targetTable.getName();
         for (Column column : targetTable.getFullSchema()) {
-            if (!column.isVisible()) {
+            if (!column.isVisible() && !column.isSequenceColumn()) {
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/UpdateCommand.java:108]: add comment to explain it

File: fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java [hunk 4]
```diff
@@ -213,7 +219,8 @@ private List<Long> bindPartitionIds(OlapTable table, List<String> partitions) {
 
     private List<Column> bindTargetColumns(OlapTable table, List<String> colsName) {
         return colsName.isEmpty()
-                ? table.getFullSchema().stream().filter(column -> column.isVisible()
+                ? table.getFullSchema().stream().filter(column -> (column.isVisible()
+                        || column.isSequenceColumn())
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/rules/analysis/BindSink.java:223]: add comment to explain why add isSequenceColumn()

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run p0

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Comment by sohardforaname** [comment]
run buildall

**Pull request closed without merging**